releases-beta/how-to-guides/schedule-a-release-cut.md

```diff
@@ -0,0 +1,57 @@
+# Scheduling a Release Cut
+
+## Creating a Release Cut
```

Comment:
This section is weird and not relevant for this doc. We should only cover the part where you say: 
 "Scheduling a Release Cut" below.

I also realized that we have an older doc for Creating scheduled releases. We should just edit that and update it with the details here.
https://docs.aviator.co/releases-beta/how-to-guides/create-a-scheduled-release